src/API/Rest/InferenceRequest.cs

```diff
@@ -232,12 +243,12 @@ public bool IsValid(out string details)
 
             if (Algorithm is null)
             {
-                errors.Add("No algorithm defined or more than one algorithms defined in 'intputResources'.  'intputResources' must include one algorithm/pipeline for the inference request.");
+                errors.Add("No algorithm defined or more than one algorithms defined in 'inputResources'.  'inputResources' must include one algorithm/pipeline for the inference request.");
             }
 
             if (InputMetadata?.Details?.Type == InferenceRequestType.DicomUid)
             {
-                if(InputMetadata.Details.Studies.IsNullOrEmpty())
+                if (InputMetadata.Details.Studies.IsNullOrEmpty())
```

Comment:
can `.Studies` be `null` here? if so, validate before dereference.

Reply:
It's a custom extension method that also checks for nulls 😁